README.md

```diff
@@ -37,8 +37,8 @@
 * Concatenate And Minify Scripts
 * Analyze JavaScript with ESLint
 * Compile ES2015
-* Notify when tasks are complete
-* Monitors change in the files and reload browser with BrowserSync
+* Notify once tasks are complete
```

Comment:
The sentence now is Notifies when tasks are complete.
Can you confirm the `once` use please.